CHANGELOG.md

```diff
@@ -1 +1,38 @@
 # Changelog
+
+Items starting with `DEPRECATE` are important deprecation notices. For more
+information on the list of deprecated flags and APIs please have a look at
+https://docs.docker.com/engine/deprecated/ where target removal dates can also
+be found.
+
+## 17.11.0-ce (2017-11-DD)
+
+IMPORTANT: Docker CE 17.11 is the first Docker release based on
```

Comment:
This should be a proper call-out. If you specifically want it to have the iconography of "important", add the Liquid thing in the code snippet below.

```markdown
> **Important**: <text>
> some more text
> some more text
{:.important}
```